### DISCUSSION TITLE
Update Readme for ValidatePattern

### DISCUSSION THREAD

**Discussion by christianmaza** [react-hook-form/react-hook-form#10611] [General]
Hello, I recently downloaded this package via NPM and found that the documentation used for the "validatePattern" rule wasn't working. After digging into the source code a little more I found that the rule should be "ValidatePattern" instead of "validatePattern". It's just a simple case change, however, I believe it would save devs some time.

**Comment by leapful** [comment]
@christianmaza 

There is no `ValidatePattern` or `validatePattern ` in the documentation of `react-hook-form`. Could you please share what source that you have found this rule ?

**Reply by christianmaza** [reply]
Sorry, I actually wrote this issue for the wrong package. I meant to write it for form-validation-react :/ Thank you for reaching back out though.
